go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	google.golang.org/grpc v1.38.0
 	k8s.io/api v0.21.2
 	k8s.io/apimachinery v0.21.2
-	k8s.io/client-go v0.21.2
+	k8s.io/client-go v12.0.0+incompatible
```

Comment:
I am not sure, but should we be concerned about this ? 
@Madhu-1 @nixpanic @humblec

Reply:
I do not think so, but we can add the v0.21.2 version in the `replace` section if needed.

Reply:
`libopenstorage/secrets` has a `replace` internally too, let's add `k8s.io/client-go => k8s.io/client-go v0.21.2` here too ?
https://github.com/libopenstorage/secrets/blob/dde442ea20ec9d59c71cea5ee0f21eeffd17ed19/go.mod#L38

Reply:
Just wanted to add this, but it already is there :rofl: See below on line 48.

Reply:
Copy that.😂🤦‍♂️